packages/block-library/src/embed/embed-preview.js

```diff
@@ -64,16 +68,27 @@ export default function EmbedPreview( {
 		className,
 		'wp-block-embed__wrapper'
 	);
+	const borderProps = useBorderProps( attributes );
 
 	// Disabled because the overlay div doesn't actually have a role or functionality
 	// as far as the user is concerned. We're just catching the first click so that
 	// the block can be selected without interacting with the embed preview that the overlay covers.
 	/* eslint-disable jsx-a11y/no-static-element-interactions */
 	const embedWrapper =
 		'wp-embed' === type ? (
-			<WpEmbedPreview html={ html } />
+			<WpEmbedPreview
+				html={ html }
+				className={ borderProps.className }
+				style={ { ...borderProps.style } }
```

Comment:
I'm not sure about these new props. Would it be more forward-compatible for them to be combined into something like `wrapperProps`? It might also help communicate the intended use for the class name and style.

It might also help get other reviewers if we can update the test instructions to cover this scenario as well as others (e.g. explaining to test with a WordPress post as an embed, youtube video, tweet, spotify link etc.) Even better if you could include some test URLs 🙏